examples/metadata/metadata-volume.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: kubernetes-metadata-volume-plugin
+  labels:
+    zone: us-est-coast
+    cluster: test-cluster1
+    rack: rack-22
+  name: kubernetes-metadata-volume-example
+  annotations:
+    build: two
+    builder: john-doe
+spec:
+  containers:
+    - name: client-container
+      image: gcr.io/google_containers/busybox
+      command: ["sh", "-c", "while true; do if [[ -e /etc/labels ]]; then cat /etc/labels; fi; if [[ -e /etc/annotations ]]; then cat /etc/annotations; fi; sleep 5; done"]
+      volumeMounts:
+        - name: podinfo
+          mountPath: /etc
+          readOnly: false
+  volumes:
+    - name: podinfo
+      metadata:
+        items:
+          - name: "labels"
+            fieldRef:
+              fieldPath: metadata.labels
+          - name: "annotations"
+            fieldRef:
+              fieldPath: metadata.annotations
```

Comment:
It doesn't have to be done in this PR, but as a follow-on, I'd like to be able to specify a specific annotation, as opposed to just the whole set.